src/Installers/Windows/build.ps1

```diff
@@ -34,7 +34,7 @@ if (-not $sharedfx64harvestroot) {
 
```

Comment:
What about src/Installers/Debian and src/Installers/Rpm❔ Might make sense to move this up into src/Installers/ and adding a build.sh there that handles Debian and Rpm

Reply:
From what I can gather, the `Debain` and `Rpm` installer projects seem to be doing their own thing. I didn't include a shell script for the Windows project because it seemed like the intention was that the project was Windows-only. Its pre-existing build script isn't consistent with the rest of the project.

As for the other ones, it's not clear to me what projects the build should be run against but let me know if you have guidance here.

Reply:
My suggestion would be to
1. Add a `build.sh` file in src/Installers/ that builds the projects under src/Installers/Debian/ and src/Installers/Rpm/ using /dockerbuild.sh Have a look at .azure/pipelines/ci.yml for the general point.
2. Move the src/Installers/Windows/build.ps1 file up into src/Installers/
3. Change build.ps1 to fix the `$repoRoot` setting and to `Push-Location $PSScriptRoot/Windows`
4. Change ci.yml to use the new src/Installers/build.ps1 location

Ideal would be to change these files to at least optionally build the shared Fx before doing the rest. But, that would be a bigger change and would mess up how ci.yml works.

Reply:
Then again, it's rare anyone actually needs to use the existing script and it's pretty annoying as-is e.g. have to build both x64 and x86 Shared Fx before using build.ps1. Let's leave this as a slight aberration in the repo.